test/libsolidity/syntaxTests/types/var_decl_val_mismatch.sol

```diff
@@ -13,5 +13,4 @@ contract n
 }
 // ----
 // SyntaxError: (14-129): No visibility specified. Did you intend to add "external"?
-// TypeError: (14-129): Fallback function must be defined as "external".
```

Comment:
Is this gone because we exit earlier?

Reply:
Yes, the order has changed due to the checks being moved from the contract level checker to the type checker (and the type checker exits early here).